src/assets/svgs/thenewboston-primary.svg

```diff
@@ -0,0 +1 @@
31 91.3928 13.7031C93.0405 13.7031 94.12 14.9247 94.1271 16.5938H88.4595ZM100.91 23H103.069L105.285 15.1236H105.448L107.664 23H109.83L113.033 12.0909H110.839L108.715 20.0668H108.609L106.478 12.0909H104.283L102.138 20.1023H102.032L99.8942 12.0909H97.6996L100.91 23ZM115.219 23H117.293V21.3026H117.471C117.854 21.9986 118.635 23.2131 120.624 23.2131C123.266 23.2131 125.184 21.0966 125.184 17.5668C125.184 14.0298 123.238 11.9489 120.603 11.9489C118.578 11.9489 117.847 13.1847 117.471 13.8594H117.343V8.45455H115.219V23ZM117.3 17.5455C117.3 15.2656 118.294 13.7528 120.148 13.7528C122.073 13.7528 123.039 15.3793 123.039 17.5455C123.039 19.733 122.044 21.402 120.148 21.402C118.323 21.402 117.3 19.8395 117.3 17.5455ZM132.163 23.2202C135.239 23.2202 137.249 20.9688 137.249 17.5952C137.249 14.2003 135.239 11.9489 132.163 11.9489C129.088 11.9489 127.078 14.2003 127.078 17.5952C127.078 20.9688 129.088 23.2202 132.163 23.2202ZM132.17 21.4375C130.161 21.4375 129.223 19.6832 129.223 17.5881C129.223 15.5 130.161 13.7244 132.17 13.7244C134.166 13.7244 135.104 15.5 135.104 17.5881C135.104 19.6832 134.166 21.4375 132.17 21.4375ZM147.794 14.7543C147.353 13.0568 146.025 11.9489 143.667 11.9489C141.203 11.9489 139.456 13.2486 139.456 15.1804C139.456 16.7287 140.393 17.7585 142.439 18.2131L144.285 18.6179C145.336 18.8523 145.826 19.321 145.826 20.0028C145.826 20.848 144.924 21.5085 143.532 21.5085C142.261 21.5085 141.444 20.9616 141.189 19.8892L139.136 20.2017C139.491 22.1335 141.096 23.2202 143.547 23.2202C146.181 23.2202 148.007 21.821 148.007 19.8466C148.007 18.3054 147.027 17.3537 145.024 16.892L143.291 16.4943C142.091 16.2102 141.572 15.8054 141.579 15.0668C141.572 14.2287 142.481 13.6321 143.689 13.6321C145.01 13.6321 145.62 14.3636 145.869 15.0952L147.794 14.7543ZM155.434 12.0909H153.197V9.47727H151.073V12.0909H149.475V13.7955H151.073V20.2372C151.066 22.2188 152.579 23.1776 154.255 23.142C154.93 23.1349 155.384 23.0071 155.633 22.9148L155.249 21.1605C155.107 21.1889 154.844 21.2528 154.504 21.2528C153.815 21.2528 153.197 21.0256 153.197 19.7969V13.7955H155.434V12.0909ZM162.261 23.2202C165.336 23.2202 167.346 20.9688 167.346 17.5952C167.346 14.2003 165.336 11.9489 162.261 11.9489C159.186 11.9489 157.176 14.2003 157.176 17.5952C157.176 20.9688 159.186 23.2202 162.261 23.2202ZM162.268 21.4375C160.258 21.4375 159.321 19.6832 159.321 17.5881C159.321 15.5 160.258 13.7244 162.268 13.7244C164.264 13.7244 165.201 15.5 165.201 17.5881C165.201 19.6832 164.264 21.4375 162.268 21.4375ZM171.84 16.5227C171.84 14.7827 172.906 13.7884 174.383 13.7884C175.825 13.7884 176.698 14.733 176.698 16.3168V23H178.822V16.0611C178.822 13.3622 177.337 11.9489 175.107 11.9489C173.467 11.9489 172.394 12.7088 171.89 13.8665H171.755V12.0909H169.717V23H171.84V16.5227Z"/><defs><linearGradient id="paint0_linear" x1="13.75" x2="5.833" y1="10.417" y2="18.333" gradientUnits="userSpaceOnUse"><stop stop-color="#B10910"/><stop offset="1" stop-color="#B10910" stop-opacity="0"/></linearGradient></defs></svg>
```

Comment:
whats the difference between this svg file and the `thenewboston.svg`? why was this one added?

Reply:
So thenewboston.svg logo only has the red icon with plain white arrow, thenewboston-primary.svg has shadow of the white arrow and thenewboston writeup in primary color.
Additionally thenewboston-white.svg has thenewboston writeup in white color.

(thenewboston.svg is to be deleted I think)

Reply:
> So thenewboston.svg logo only has the red icon with plain white arrow, thenewboston-primary.svg has shadow of the white arrow and thenewboston writeup in primary color.
> Additionally thenewboston-white.svg has thenewboston writeup in white color.
> 
> (thenewboston.svg is to be deleted I think)

Correct! The logo with the shadow is the correct logo. Thank you!

Reply:
Correct! The logo with the shadow is the correct logo. Thank you!